.github/workflows/build_and_test_with_worker_events.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        openresty-version: [1.19.9.1, 1.21.4.1]
+        openresty-version: [1.21.4.1]
```

Comment:
Not a blocker for this PR, just a note for the feature: it's better to have different PRs for unrelated changes.